controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -569,12 +564,7 @@ public OperationContext createContext(String scope, String name) {
                                                           final UUID txId, final OperationContext context,
                                                           final Executor executor) {
         Stream stream = getStream(scope, streamName, context);
-        return withCompletion(stream.commitTransaction(txId), executor)
-                .thenApply(result -> {
-                    stream.getNumberOfOngoingTransactions().thenAccept(count ->
-                            TransactionMetrics.reportOpenTransactions(scope, streamName, count));
-                    return result;
-                });
+        return withCompletion(stream.commitTransaction(txId), executor);
```

Comment:
You may want to check the impact of removing this with the platform team. Just wondering if they use this metric extensively, removing it without an alternative may might be an issue for them.

Reply:
I have moved it to a different point where it is computed. 
After completion of a commit workflow. 
Look at persistentStreamBase